tests/handlers/test_appservice.py

```diff
@@ -252,6 +253,38 @@ async def get_3pe_protocol(service, unusedProtocol):
             },
         )
 
+    def test_notify_interested_services_ephemeral(self):
```

Comment:
Could you add docstrings to these functions? Each with a sentence or two of what part of the `notify_interested_services_ephemeral` behaviour they test?

Reply:
https://github.com/matrix-org/synapse/pull/11207/commits/451c31f67a5e37da3d5afdfabe618f50e08edcab